### ISSUE TITLE
Initial seeder / fixtures

### ISSUE THREAD

**Issue by divagueame** [ShortRuby/rubyandrails.info#47]
Hi,
I'm opening this issue to discuss about possibly adding some seeder or fixtures so that people who'd like to contribute can get a local environment closer to the actual website.

**Comment by lucianghinda** [comment]
@divagueame yes, this would be an awesome initial experience for someone setting up the project first time

**Comment by divagueame** [comment]
This can be a starting point:
https://github.com/ShortRuby/rubyandrails.info/pull/55
